qacode/core/utils/Utils.py

```diff
@@ -0,0 +1,28 @@
+from sys import version_info
+from os import path
+
+class Utils(object):
```

Comment:
Why the class? Why not use just module-level methods? This is not Java 😆 .

In Python, module-level methods are preferred over a class with only static methods.

Reply:
starting to doing this.. !!